spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/support/SpringMvcContract.java

```diff
@@ -64,6 +69,7 @@
 
 	private final ConversionService conversionService;
 	private final Param.Expander expander;
+	private ResourceLoader resourceLoader = new DefaultResourceLoader();
```

Comment:
This class implements ResourceLoaderAware and therefore has a setResourceLoader() method. You are suggesting I remove the interface and instead that I simply use `	@Autowired
	private ApplicationContext applicationContext;` instead correct?

The reason I went this route is because I was bit concerned about this class being used without and application context around and the field initialization was meant to guard against a possible NPE as a result. That said, confirm the change and I'll update the code.